src/dxgi/dxgi_swapchain.cpp

```diff
@@ -731,7 +737,13 @@ namespace dxvk {
       SetGammaControl(0, nullptr);
       ReleaseMonitorData();
     }
-    
+
+    if (m_ModeChangeInProgress) {
```

Comment:
Is it intended that this check happens **after** changing the display mode and messing with gamma?

Reply:
Yes, indeed, that makes no sense. My intention was to leave some error checking in place (while leaving display mode and gamma behind is just a bug). But this error checking probably doesn't make much sense anyway for recursive invocation, I moved the recursion check to the beginning of the functions.